content/articles/2017-11-01-lubridate-1.7.0.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+title: lubridate 1.7.0
+slug: lubridate-1.7.0
+description: >
+  A new version of lubridate is on CRAN!
+author: Vitalie Spinu
+date: 2017-11-01
+photo:
+  url: https://unsplash.com/photos/yBzrPGLjMQw
+  author: Heather Zabriskie
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library("lubridate")
+knitr::opts_chunk$set(collapse = TRUE, comment = "##=>")
+```
+
+We are pleased to announce that [lubridate
+1.7.0](https://cran.r-project.org/web/packages/lubridate/index.html) is now on
+CRAN! For a complete set of changes please see the release
+[NEWS](https://github.com/tidyverse/lubridate/releases/tag/v1.7.0).
+
+Functions to work with date-times and time-spans: fast and user friendly parsing
+of date-time data, extraction and updating of components of a date-time (years,
+months, days, hours, minutes, and seconds), algebraic manipulation on date-time
+and time-span objects.
+
+Lubridate is a package that makes working with date-time and time-span objects
```

Comment:
I think you accidentally duplicated this paragraph. Otherwise, it's good to go!

Reply:
Arh, sorry about that. Fixed. 

BTW, should titles be capitalized even if it's a package name? If the package name starts a sentence it would be capitalized. I think same should hold for titles, even when it's as short as "Lubridate 1.7.0". It  looks nicer this way on top of the page.

Reply:
I don't have strong feelings either way, but to date we've treated package names like actual names and preserved capitalisation.